src/app/backend/replicasetpods.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+// Information about a Container that belongs to a Pod.
+type PodContainer struct {
+	// Name of a Container.
+	Name string `json:"name"`
+
+	// Container state.
+	State string `json:"state"`
+
+	// Number of restarts.
+	RestartCount int `json:"restartCount"`
+}
+
+// List of pods that belongs to a Replica Set.
+type ReplicaSetPods struct {
+	// List of pods that belongs to a Replica Set.
+	Pods []ReplicaSetPodWithContainers `json:"pods"`
+}
+
+// Detailed information about a Pod that belongs to a Replica Set.
+type ReplicaSetPodWithContainers struct {
+	// Name of the Pod.
+	Name string `json:"name"`
+
+	// Time the Pod has started. Empty if not started.
+	StartTime *unversioned.Time `json:"startTime"`
+
+	// List of Containers that belongs to particular Pod.
+	PodContainers []PodContainer `json:"podContainers"`
+}
+
+// Returns list of pods with containers for the given replica set in the given namespace.
+func GetReplicaSetPods(client *client.Client, namespace string, name string) (
+	*ReplicaSetPods, error) {
+
+	replicaSetPods := &ReplicaSetPods{}
+
+	replicaSet, err := client.ReplicationControllers(namespace).Get(name)
+	if err != nil {
+		return nil, err
+	}
+
+	labelSelector := labels.SelectorFromSet(replicaSet.Spec.Selector)
```

Comment:
Can you share the code that gets list of pods for a replica set with replicasetdetails.go? Maybe extract to a separate file or something